src/main/java/com/yureto/supergt/SuperGtController.java

```diff
@@ -41,4 +42,15 @@ public ResponseEntity<SuperGtResponse> insert(@RequestBody @Validated SuperGtReq
         SuperGtResponse message = new SuperGtResponse("new driver created");
         return ResponseEntity.created(uri).body(message);
     }
+
+    @PatchMapping("/superGt/{id}")
+    public ResponseEntity<SuperGtResponse> update(@PathVariable("id") Integer id, @RequestBody @Validated SuperGtRequest superGtRequest) {
+        SuperGt superGt = superGtService.findById(id);
+        superGt.setDriver(superGtRequest.getDriver());
+        superGt.setAffiliatedTeam(superGtRequest.getAffiliatedTeam());
+        superGt.setCarNumber(superGtRequest.getCarNumber());
+        superGtService.update(id, superGtRequest);
+        SuperGtResponse message = new SuperGtResponse("driver updated");
+        return new ResponseEntity<>(message, HttpStatus.OK);
```

Comment:
GETの方もですが、以下の書き方の方が簡潔で良さそうです。
```suggestion
        return new ResponseEntity.ok(message);
```